Sources/Core/Extensions/Foundation/DateFormatter+PovioKit.swift

```diff
@@ -0,0 +1,91 @@
+//
+//  DateFormatter+PovioKit.swift
+//  PovioKit
+//
+//  Created by Borut Tomazin on 02/10/2024.
+//  Copyright © 2024 Povio Inc. All rights reserved.
+//
+
+import Foundation
+
+extension DateFormatter {
+  /// Format a time in a 12-hour format with AM/PM designation.
+  ///
+  /// `9:41 PM`
+  static let time12Hour: DateFormatter = {
+    let dateFormatter = DateFormatter()
+    dateFormatter.locale = .autoupdatingCurrent
+    dateFormatter.dateFormat = "h:mm a"
+    return dateFormatter
+  }()
+  
+  /// Format a time in a 24-hour format without AM/PM designation.
+  ///
+  /// `9:41`
+  static let time24Hour: DateFormatter = {
+    let dateFormatter = DateFormatter()
+    dateFormatter.locale = .autoupdatingCurrent
+    dateFormatter.dateFormat = "h:mm"
```

Comment:
I'm guessing we'd want `HH:mm` format here otherwise if it's 15:03 it translates to 3:03 and we don't know if it's AM or PM

Reply:
Thanks for the notice. Your solution is indeed better. But I would only have one Hour component to not show leading zeroes.